src/utils.mjs

```diff
@@ -51,6 +51,35 @@ export function getMaskedTime(timePadding) {
   return maskTime(Date.now(), timePadding);
 }
 
+/**
+ * Extract the OS from the user agent string
+ * @returns {Enumerator(':android', ':ios', ':ipados', '')} the OS
+ */
+function getMobileOS(userAgent) {
+  if (userAgent.includes('android')) {
+    return ':android';
+  } else if (userAgent.includes('ipad')) {
```

Comment:
FWIW, this is outdated. iPad has reported as desktop for years now.

There is a toggle, but it's not the default and should not be relied upon.

https://forums.developer.apple.com/forums/thread/119186